### PR TITLE
1.1.2

### PR DESCRIPTION
**Pull request by ghermans**

### CODE REVIEW

File: vf_convenience/cl_menu.lua [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+function CreateStoreMenu(menu)  
+  inventoryItems = exports.vf_base:GetInventory()
+  ConvenienceStore = {
+    {name = "SNK_ITEM1", description = "SNK_ITEM1_D", price = 10},
+    {name = "SNK_ITEM2", description = "SNK_ITEM2_D", price = 20},
+    {name = "SNK_ITEM3", description = "SNK_ITEM3_D", price = 10},
+    {name = "SNK_ITEM4", description = "SNK_ITEM4_D", price = 30},
+    {name = "SNK_ITEM5", description = "SNK_ITEM5_D", price = 50},
+    {name = "SNK_ITEM6", description = "SNK_ITEM6_D", price = 50} 
+  }
```
Comment (pongo1231) [inline vf_convenience/cl_menu.lua]: Maybe move these to config.lua also so users can easily customize them.

File: vf_convenience/cl_menu.lua [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    while not HasThisAdditionalTextLoaded(gxt, CurrentSlot) do
+      Citizen.Wait(0)
+    end
+  end
+
+  Citizen.Wait(1800)
+end)
+
+function CreateStoreMenu(menu)  
+  inventoryItems = exports.vf_base:GetInventory()
```
Comment (pongo1231) [inline vf_convenience/cl_menu.lua]: It's a good practice to define variables as local as long as you don't need them outside that scope so they don't leak over accidentally.

File: vf_convenience/client.lua [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+Citizen.CreateThread(function()
+  DestroyAllCams(0)
+  Wait(5000)
+
+  while true do
+    Wait(10)
+      if IsNearStore then
+        if IsStoreKeeperAlive then
+          if IsPlayerNearShopKeeper(storeKeeper) and not IsPedDeadOrDying(storeKeeper) then
```
Comment (pongo1231) [inline vf_convenience/client.lua]: Minor suggestion but these could be fused together to 1 if statement unless I'm missing something.

File: vf_convenience/client.lua [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+      if NoCops then
+        if GetClockHours() == 07 then
+          speech = "SHOP_NO_COPS_START"
+        elseif GetClockHours() >= 09 and GetClockHours() <= 21 then 
+          speech = "SHOP_NO_ENTRY"
+        else
+          speech = "SHOP_NO_COPS_END"
+        end
+      else
+        speech ""
```
Comment (pongo1231) [inline vf_convenience/client.lua]: Syntax error (should be `speech = ""`)

File: vf_convenience/cl_menu.lua (hunk 1 above)
Comment (ghermans) [reply vf_convenience/cl_menu.lua]: if i remember correctly this is using labels so adding custom product names would probably return a a NULL response.

**Pull request merged**